pandas/core/dtypes/dtypes.py

```diff
@@ -999,7 +999,8 @@ def __str__(self) -> str_type:
 
     @property
     def name(self) -> str_type:
-        return f"period[{self._freqstr}]"
+        dname = {"ME": "period[M]"}
+        return dname.get(self._freqstr, f"period[{self._freqstr}]")
```

Comment:
we can probably use the new function here too?

Reply:
Yes, you are right. I refactored this function.